thirdparty/README.md

```diff
@@ -386,6 +386,17 @@ Files extracted from upstream source:
 - `COPYING`
 
 
+## grisu2
```

Comment:
Should be mentioned in the [COPYRIGHT.txt](https://github.com/godotengine/godot/blob/master/COPYRIGHT.txt) as well.